docker-service.sh

```diff
@@ -32,16 +32,15 @@
 #
 # wsl -d Ubuntu ~/bin/docker-service Ubuntu
 
-[ -z $1 ] && DOCKER_DISTRO="" || DOCKER_DISTRO="-d $1"
+[ -z "$1" ] && DOCKER_DISTRO="$WSL_DISTRO_NAME" || DOCKER_DISTRO="$1"
```

Comment:
I also changed this so now it is enclosed in double quotes. I don't really know the difference. Let me know your feedback.